cli/lib/kontena/machine/aws/node_provisioner.rb

```diff
@@ -96,21 +98,18 @@ def run!(opts)
 
         ##
         # @param [String] grid
-        # @return [Aws::EC2::SecurityGroup]
+        # @return [Array] Security group id in array
```

Comment:
ok, maybe the method name is a bit misleading now.. but this is not a blocker for this pr :)